packages-common/pluggable-widgets-generator/generators/app/templates/pluggable/native/fullTemplateTs/src/WidgetName.tsx.ejs

```diff
@@ -0,0 +1,30 @@
+import { Component, ReactNode, createElement } from "react";
+import { TextStyle, ViewStyle } from "react-native";
+import { BadgeSample } from "./components/BadgeSample";
+import { <%= widgetName %>Props } from "../typings/<%= widgetName %>Props";
+import { Style } from "./utils/common";
+
+export interface BadgeStyle extends Style {
+    container: ViewStyle;
+    badge: ViewStyle;
+    label: TextStyle;
+}
+
+export class <%= widgetName %> extends Component<<%= widgetName %>Props<BadgeStyle>> {
+    private readonly onClickHandler = this.onClick.bind(this);
+
+    render(): ReactNode {
+        return (
+            <BadgeSample
+                style={this.props.style}
+                onClickAction={this.onClickHandler}
+                value={this.props.valueAttribute ? this.props.valueAttribute.displayValue : "Default"} />
```

Comment:
?? "Default"

Reply:
Changed to: `value={this.props.value?.displayValue || "Default"}`
`displayValue` is always a string (never undefined), so the ?? operator won't work.